documentation/modules/changing-precopy-intervals-cli.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="changing-precopy-intervals-cli_{context}"]
+= Changing precopy intervals for warm migration
+
+The snapshots for the precopy stage of warm migration are created at one-hour intervals by default.
+
+You can change the snapshot interval by editing the `forklift-controller` deployment.
+
+.Prerequisites
+
+* You must enable link:https://kb.vmware.com/s/article/1020128[changed block tracking (CBT)] on the VMware virtual machines and virtual disks.
```

Comment:
This is a general prerequisite in order to be able to warm migrate a VMware VM, not specific to change the interval time between precopies

Reply:
oh I see now that there is no dedicated section for warm migration.

Reply:
I have not yet decided where this module is going to go.